### PR TITLE
support config to customize refresh executor

### PR DESCRIPTION
**Pull request by HzjNeverStop**

### CODE REVIEW

File: sofa-boot-project/sofa-boot-core/isle-sofa-boot/src/main/java/com/alipay/sofa/isle/spring/config/SofaModuleProperties.java [hunk 1]
```diff
@@ -101,4 +104,28 @@ public boolean isIgnoreModuleInstallFailure() {
     public void setIgnoreModuleInstallFailure(boolean ignoreModuleInstallFailure) {
         this.ignoreModuleInstallFailure = ignoreModuleInstallFailure;
     }
+
+    public float getParallelRefreshCoreCountMagnifier() {
+        return parallelRefreshCoreCountMagnifier;
```
Comment (ujjboy) [inline sofa-boot-project/sofa-boot-core/isle-sofa-boot/src/main/java/com/alipay/sofa/isle/spring/config/SofaModuleProperties.java:109]: Magnifier  这个单词换一下   是说一个倍数吗？   可以用 factor

**Review by ujjboy — CHANGES_REQUESTED** [review]

File: sofa-boot-project/sofa-boot-core/isle-sofa-boot/src/main/java/com/alipay/sofa/isle/spring/config/SofaModuleProperties.java (hunk 1 above)
Comment (HzjNeverStop) [reply sofa-boot-project/sofa-boot-core/isle-sofa-boot/src/main/java/com/alipay/sofa/isle/spring/config/SofaModuleProperties.java]: 已修改

**Review by ujjboy — APPROVED** [review]
LGTM

**Comment by ujjboy** [comment]
Please commit your change before run this shell, un commit files:
 M sofa-boot-project/sofa-boot-core/startup-sofa-boot/src/main/java/com/alipay/sofa/startup/stage/StartupContextRefreshedListener.java
./tools/check_format.sh: 11: exit: Illegal number: -1

**Pull request merged**